config/backends.go

```diff
@@ -75,13 +75,19 @@ func (cfg *Aerospike) validateAndLog() error {
 }
 
 type Cassandra struct {
-	Hosts    string `mapstructure:"hosts"`
-	Keyspace string `mapstructure:"keyspace"`
+	Hosts      string `mapstructure:"hosts"`
+	Keyspace   string `mapstructure:"keyspace"`
+	DefaultTTL int    `mapstructure:"default_ttl_seconds"`
 }
 
 func (cfg *Cassandra) validateAndLog() error {
 	log.Infof("config.backend.cassandra.hosts: %s", cfg.Hosts)
 	log.Infof("config.backend.cassandra.keyspace: %s", cfg.Keyspace)
+	if cfg.DefaultTTL <= 0 {
+		cfg.DefaultTTL = 2400
```

Comment:
> Shouldn't this default TTL assignment be set in setConfigDefaults in config/config.go: v.SetDefault("backend.cassandra.default_ttl_seconds", 2400)?

Added. I kept this check though because I think it still represents the logic we have in place in prod from pull request #89 on. Basically the description of the `getMaxTTLSeconds(cfg config.Configuration)` function written found `backends/config/config.go` better describes what we are trying to do.

Thoughts?